boot/main.go

```diff
@@ -1,6 +1,7 @@
 package boot
 
 import (
+	"github.com/miekg/dns"
```

Comment:
NIT: I prefer external imports to happen at the endof the import block.

Reply:
Doesn't gofmt determine the order?